site/content/docs/commands/job-run.ja.md

```diff
@@ -0,0 +1,24 @@
+# job run
+```bash
+$ copilot job run
+```
+
+## コマンドの概要
+
+`copilot job run` は、スケジュールされた Job を実行します。
+
+## フラグ
+```bash
```

Comment:
nit: We need to change this in the English version-- sorry. If we use `console` instead of `bash`, the colors will be uniform. 
```suggestion
```console
```